lib/Dashboard/CalendarWidget.php

```diff
@@ -25,41 +25,52 @@
 
 namespace OCA\Calendar\Dashboard;
 
+use DateInterval;
+use DateTime;
+use DateTimeImmutable;
 use OCA\Calendar\AppInfo\Application;
 use OCA\Calendar\Service\JSDataService;
-use OCP\Dashboard\IWidget;
-use OCP\IInitialStateService;
+use OCA\DAV\CalDAV\CalDavBackend;
```

Comment:
private API of another app. we should be able to go through `\OCP\Calendar\IManager::searchForPrincipal`

Reply:
@miaulalala why collapse if this hasn't been addressed?

Reply:
sorry, didn't push the changes but had it done locally already. it's done now